server/sample.env

```diff
@@ -1,3 +1,3 @@
 JWT_SECRET=
-MONGO_URI=
+MONGO_URI=mongodb://localhost/
```

Comment:
This isn't really necessary change.

Reply:
sorry, I probably initially pasted to the exapmle env instead of my env and didn't pay attention